src/olympia/addons/tests/test_admin.py

```diff
@@ -140,6 +140,45 @@ def test_can_edit_with_addons_edit_permission(self):
         addon.reload()
         assert addon.guid == '@bar'
 
+    def test_show_link_to_reviewer_tools_listed(self):
+        addon = addon_factory(guid='@foo')
+        version_factory(addon=addon, channel=amo.RELEASE_CHANNEL_LISTED)
+        detail_url = reverse('admin:addons_addon_change', args=(addon.pk,))
+        user = user_factory()
+        self.grant_permission(user, 'Admin:Tools')
+        self.grant_permission(user, 'Addons:Edit')
+        self.client.login(email=user.email)
+        response = self.client.get(detail_url, follow=True)
+        assert 'Reviewer Tools (listed)' in response.content.decode('utf-8')
+        assert ('Reviewer Tools (unlisted)' not in
+                response.content.decode('utf-8'))
```

Comment:
you can either `content = response.content.decode('utf-8')` once or test with `b'...'` strings so you don't have to `.decode(...)` each time.